pkgs/tools/package-management/nix/common.nix

```diff
@@ -3,7 +3,7 @@
 , version
 , suffix ? ""
 , hash ? null
-, src ? fetchFromGitHub { owner = "NixOS"; repo = "nix"; rev = version; inherit hash; }
+, src ? fetchFromGitHub { owner = "auxolotl"; repo = "nix"; rev = version; inherit hash; }
```

Comment:
I hope I can look back on this change in a few years and be as excited as I am right now.